devtools-ci-index.yaml

```diff
@@ -3331,12 +3315,12 @@
             ci_cmd: '/bin/bash cico_build.sh'
             timeout: '20m'
             discarder_days: 30
-        - '{ci_project}-{git_repo}-build-master-update-downstream':
+        - '{ci_project}-{git_repo}-build-master':
```

Comment:
'{ci_project}-{git_repo}-build-master' template missing '*fabric8-hub-token' secret. So it won't work as '{ci_project}-{git_repo}-build-master-update-downstream'

Reply:
@alexeykazakov `build-master-update-downstream` template was used for f8-xxx-client projects which anyway not required `fabric8-hub-token` as these projects builds are not doing any github_push.  This was discussed with @dipak-pawar and agreed.